documentation/docs/pages/operators/nodes/preliminary-steps/vps-setup/advanced.mdx

```diff
@@ -0,0 +1,533 @@
+
+import { Callout } from 'nextra/components';
+import { Tabs } from 'nextra/components';
+import { VarInfo } from 'components/variable-info.tsx';
+import { Steps } from 'nextra/components';
+import {Accordion, AccordionItem} from "@nextui-org/react";
+import { MyTab } from 'components/generic-tabs.tsx';
+import { AccordionTemplate } from 'components/accordion-template.tsx';
+
+# Advanced Server Administration
+
+This page is for experienced operators and aspiring sys-admins who seek for higher optimisation and better efficiency of their work managing Nym infrastructure. The steps shared on this page cannot be simply copy-pasted, they ask you for more attention and consideration all the way from choosing server and OS to specs per VM allocation.
+
+<VarInfo />
+
+## Virtualising a Dedicated Server
+
+Some operators or squads of operators orchestrate multiple Nym nodes. Among other benefits (which are out of scope of this page), these operators can decide to acquire one larger dedicated (or bare-metal) server with enough specs (CPU, RAM, storage, bandwidth and port speed) to meet [minimum requirements](../../nodes#minimum-requirements) for multiple nodes run in parallel.
+
+This guide explains how to prepare your server in order to be able to host multiple nodes running on separated VMs.
+
+<Callout type="info">
+This guide is based on Ubuntu 22.04, in case you prefer another OS, you may have to do a bit of your own research to troubleshoot networking configuration and other parameters.
+</Callout>
+
+### Installing KVM on a Server with Ubuntu 22.04
+
+**KVM** stands for **Kernel-based Virtual Machine**. It is a virtualization technology for Linux that allows a user to run multiple virtual machines (VMs) on a single physical machine. KVM turns the Linux kernel into a hypervisor, enabling it to manage multiple virtualised systems.
+
+Follow the steps below to install KVM on Ubuntu 22.04 LTS:
+
+#### Prerequisites
+
+<Callout type="warning">
```

Comment:
All these links are 404-ing in the preview for me. 

Typo in `Additionaly` (should be `Additionally`)